FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -122,17 +123,52 @@ - (NSString *)snapshotVerifyViewOrLayer:(id)viewOrLayer
       }
     }
   }
-  
-  if (!testSuccess) {
-    return [NSString stringWithFormat:@"Snapshot comparison failed: %@", errors.firstObject];
-  }
+    
+  [self addAttachmentsWithErrors:errors identifier:identifier];
+    
   if (self.recordMode) {
-    return @"Test ran in record mode. Reference image is now saved. Disable record mode to perform an actual snapshot comparison!";
+    if (errors.count > 0) {
+      return [NSString stringWithFormat:@"Snapshot comparison failed: %@", errors.firstObject];
```

Comment:
Is this possible? Looking at L102-L105, it is doing this:

```
     BOOL referenceImageSaved = [self _compareSnapshotOfViewOrLayer:viewOrLayer referenceImagesDirectory:referenceImagesDirectory identifier:(identifier) tolerance:tolerance error:&error];
     if (!referenceImageSaved) {
       [errors addObject:error];
     }
```

When `self.recordMode` is on, `compareSnapshotOfViewOrLayer:...` the BOOL from the function relates to whether or not the reference image was saved.